### PR TITLE
updated OWNER_ALIASES to reflect state of roles

### PR DESCRIPTION
**Pull request by dofinn**
This PR updates OWNER_ALIASES to reflect true state of roles within teams. Engineers whom have left team/RH have also been removed.

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -5,28 +5,26 @@ aliases:
...
     - fahlmant
     - jaybeeunix
-    - jharrington22
     - mrbarge
-    - NautiluX
     - rafael-azevedo
-    - rogbas
     - wanghaoran1988
     - blrm
+	- iamkirkbater
```
Comment (mrbarge) [inline OWNERS_ALIASES:12]: Bad indenting here?

Comment (cblecker) [inline OWNERS_ALIASES:12]:
```suggestion
    - iamkirkbater
```

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -5,28 +5,26 @@ aliases:
     - dustman9000
     - fahlmant
     - jaybeeunix
```
Comment (cblecker) [inline OWNERS_ALIASES:7]:
```suggestion
```

**Pull request merged**